modules/Poco/Foundation.cppm

```diff
@@ -262,7 +262,6 @@ module;
 #include "Poco/Timezone.h"
 #include "Poco/Token.h"
 #include "Poco/Tuple.h"
-#include "Poco/TypeList.h"
```

Comment:
Shouldn't there be a `POCO_TYPELIST_INTERNAL` definition before this inclusion? 
Also (unrelated), some headers are conditionally included but their `using` counterparts are there unconditionally.

Reply:
Include is actually removed but the question triggered missing changes in modules to correct it.